### PR TITLE
EWL-3873 SG | Refactor patterns that use the "Two-up-heavy-left" layout and remove the layout molecule

### PR DESCRIPTION
**Pull request by chelsiejohnston**
## Ticket(s)

**Jira Ticket**

- [EWL-3873: SG | Refactor patterns that use the "Two-up-heavy-left" layout and remove the layout molecule](https://issues.ama-assn.org/browse/EWL-3873)

## Description

- Adds `.col-width-4` and `.col-width-8` classes to `_layouts.scss`
- Refactors `molecules-featured-content-hero-row`, `molecules-news-item-standard-row`, `organisms-search_modal` to use `.col-width-x` classes
- Removes `molecules-layout-two-up-heavy-left` pattern

## To Test

- [x] **make sure you have the correct branch checked out:** `feature/EWL-3873-refactor-two-up-heavier-left` (NOT `feature/EWL-3872-refactor-two-up-heavier-left-layout`). I named this branch after the wrong layout. Sorry!
- [ ] Review all Template patterns and make sure they don't look broken
- [ ] Review molecules-featured-content-hero-row, molecules-list-featured-content-hero, organisms-audience-featured-hero and make sure they don't look broken
- [ ] Review molecules-news-item-standard-row, organisms-news-standard-list
- [ ] Review organisms-search_modal and make sure they don't look broken
- [x] Molecules > Layouts: verify that the "Two Up Heavy Left" layout has been removed from the list of patterns

## Relevant Screenshots/GIFs

N/A

## Remaining Tasks

N/A

## Additional Notes

N/A

### CODE REVIEW

**Comment by rpama** [comment]
`news-item-standard-row` and `featured-content-hero-row` have some formatting issues at screen sizes between 740px and 900px.

screen shot 2017-08-25 at 9 27 42 am

The `Home` templated likewise has issues because it contains the `featured-content-hero-row` component.

**Comment by rpama** [comment]
I was wrong about this being good to go (shame on me for just trying to get the review in quickly). 

- [ ] There's some inconsistent left and right margin/padding between the breakpoints. The added margins/padding are a regression from the original pattern (which maybe is ok). Whatever behavior we go with though, the application or not of margins and padding should be consistent across the breakpoints.

row-margins-disappear

Applies to `molecules-featured-content-hero-row`, `molecules-news-item-standard-row`

- [ ] The `layout` class had a max width of 1260px. This seems to be lost from the `molecules-list-featured-content-hero`, `organisms-audience-featured-hero` pattern now. I don't know if this was a conscious decision and we know we're baking in the max widths at a higher level, or if this was just missed in the refactor.

- [ ] And the `organisms-search_modal` mobile view breakpoint also needs to be 740px and not 900px to be at parity with the current implementation.

**Comment by rpama** [comment]
@chelsiejohnston I've pushed some commits of additional clean up. If could take a look at these to ensure you have no major objections, then this will be good to merge.

And I apologize for just pushing commits, but I don't want you to blocked by these issues anymore. Not to mention it seemed it would be more tedious for you for me to pass them back rather than just make the tweaks myself. You did a lot of great work on this and 201!!

File: styleguide/source/_patterns/01-molecules/06-components/07-featured-content-hero-row/_featured-content-hero-row.scss [hunk 1]
```diff
@@ -44,15 +44,13 @@
...
+    
     img {
       float: right;
       max-height: 100%;
       max-width: 100%;
       width: initial;
     }
   }
-
-  @include breakpoint($bp-small) {
```
Comment (chelsiejohnston) [inline styleguide/source/_patterns/01-molecules/06-components/07-featured-content-hero-row/_featured-content-hero-row.scss]: @rpama please move the breakpoint declaration above the `img` styling - we should be grouping includes before nested elements.

File: styleguide/source/_patterns/03-templates/01-home/_01-home.scss [hunk 2]
```diff
@@ -1,8 +1,18 @@
 .layout-home {
+  padding: 0 20px;
 
   @include breakpoint($bp-med-home min-width) {
     padding: 0 30px;
   }
+
+  .col-width-3,
```
Comment (chelsiejohnston) [inline styleguide/source/_patterns/03-templates/01-home/_01-home.scss]: @rpama I'm not sure these changes were necessary, but I'm fine with merging them and then removing them later once #201 is merged.

**Comment by chelsiejohnston** [comment]
@rpama one change for you in code comments, and another general comment. Otherwise, LGTM

File: styleguide/source/_patterns/03-templates/01-home/_01-home.scss (hunk 2 above)
Comment (rpama) [reply styleguide/source/_patterns/03-templates/01-home/_01-home.scss]: So, if I recall correctly from Friday, the news_item stuff was getting some extra spacing on the home page and it bugged me so I decided this was a the easiest way to fix it (although not entirely elegant).

**Comment by chelsiejohnston** [comment]
bitmoji

**Pull request merged**